guide/getting-started/software.md

```diff
@@ -36,11 +36,13 @@ Illustration sources
 
 # Software overview
 
-Finance is complex, and there are many use cases, from giving weekly allowance to a child and selling goods in a store to managing multi-billion dollar retirement funds and hectic day trading. So we end up with many different application solutions for the various use cases. Bitcoin being a global currency adds another layer of complexity as the software may need to be adapted to different languages, cultures, and legal frameworks. When designing, it's a good idea to feel for this landscape and know where your product fits. Here is an overview of several common and specialized categories.
+There are countless use cases for financial tools, so there are almost as many application solutions per use case. Bitcoin’s global functionality adds complexity that may require adapting your application to foreign languages, cultures, and legal frameworks.
```

Comment:
Adding comment by @sqcken for future reference:

> I chopped this down significantly. We need fewer word meatloafs. Is there a way to clear it up?

Reply:
What does he mean by `Is there a way to clear it up?`?

Taking another look at the page content, I'd change this intro now. The page is not just about financial tools as it includes miners and nodes. It's more that the unique architecture of Bitcoin has resulted in several categories of software. Some are for interacting with the currency (wallets) and others for interacting with the network (nodes, miners). CLIs are a text-based UI, contrasted with a GUI. Maybe all we need is something simple like this:
`The unique history and architecture of Bitcoin has resulted in a few distinct categories of software, with each one serving a distinct purpose.`

Reply:
Replaced the intro https://github.com/BitcoinDesign/Guide/pull/437/commits/2a8065210dd4b367035f82cd0cbc35e444b39a95